akka-docs/rst/java/code/docs/stream/StreamTestKitDocTest.java

```diff
@@ -120,11 +120,11 @@ public void sinkActorRef() throws Exception {
     final TestProbe probe = new TestProbe(system);
     final Cancellable cancellable = sourceUnderTest
       .to(Sink.actorRef(probe.ref(), Tick.COMPLETED)).run(mat);
-    probe.expectMsg(Duration.create(1, TimeUnit.SECONDS), Tick.TOCK);
+    probe.expectMsg(Duration.create(3, TimeUnit.SECONDS), Tick.TOCK);
     probe.expectNoMsg(Duration.create(100, TimeUnit.MILLISECONDS));
```

Comment:
I'm staring at this and can't see why there couldn't be another TOCK in flight here. Is that because things are fused? Is that what we want to illustrate in this doc snippet?